worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -146,7 +147,10 @@ void processContract(Contract contract, Job job, int month, Path outputDirPath,
                 progressTracker.getFailureCount()));
     }
 
-    void processContractBenes(Job job, Contract contract, int month, ProgressTracker progressTracker) throws ExecutionException, InterruptedException {
+    void processContractBenes(Job job, int month, ProgressTracker progressTracker)
+            throws ExecutionException, InterruptedException {
+        Contract contract = job.getContract();
+        assert contract != null;
```

Comment:
Another assert in here

Reply:
Asserts are intentional.  With the new assumption that a User is bound to a single Contract, results in a Job being bound to a single contract.

The contract reference must exist.